nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/PutElasticsearchHttp.java

```diff
@@ -321,9 +321,16 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             final Response getResponse;
             try {
                 getResponse = sendRequestToElasticsearch(okHttpClient, url, username, password, "PUT", requestBody);
-            } catch (IllegalStateException | IOException ioe) {
-                throw new ProcessException(ioe);
+            } catch (final Exception e) {
```

Comment:
Will there be any exceptions encountered here where we would want to route to RETRY instead?